src/util/token.ts

```diff
@@ -0,0 +1,21 @@
+import { Issuer } from "../types";
+import { contractInstance } from "../common/smartContract/contractInstance";
+
+const getContractInstance = (issuers: Issuer[]) => {
```

Comment:
If it's only one issuer, then isn't it just `issuers[0].tokenRegistry`, error otherwise?